src/util/dlist/dlist_debug.h

```diff
@@ -136,6 +138,8 @@ static inline void dlist_add_prev(struct dlist_head *_new,
 	 * initialization */
 	// assert(__is_linked(list)); /* add to not initialized list */
 
+	// __dlist_debug_check(_new);
```

Comment:
This 2 checks commented because otherwise system goes crazy during tests and hangs on infinite loop. Don't know why.

Reply:
First check causes error because `_new` is not supposed to be initialized, so generally it's zeroes or garbage there. All fields are set later in `_dlist_add`